docs/api.md

```diff
@@ -0,0 +1,41 @@
+<!-- TOC -->
+
+- [1. API Documentation](#1-api-documentation)
+  - [1.1. POST /api/tests](#14-post-apitests)
+
+## 1.1. POST /api/tests
+
+Creates a synthetic test
+
+### 1.1.1. Expected Payload
+
+```json
+{
+  "test": {
+    "title": "My new test",
+    "locations": ["us-east-1"],
+    "minutesBetweenRuns": 60,
+    "type": "API",
+    "http_request": {
+      "http_method": "GET",
```

Comment:
This naming feels redundant (we know it's an http method since it's in the http_request property), but left it this way since it's what the current `testRunnerV0` expects. Trello task to change testRunner: https://trello.com/c/JlISZjNO/50-change-test-runner-v0-event-shape